### ISSUE TITLE
Switching from Plugins page on simple site to atomic site with Jetpack Beta Test enabled will append query string of site title

### ISSUE THREAD

**Issue by jjchrisdiehl** [Automattic/wp-calypso#49576]
### Steps to reproduce the behavior

1. Go to Simple site > Plugins
2. Switch Sites to an atomic site with Jetpack Beta Test enabled
3. This will load the plugins page on the atomic site but it will append a query string searching for a plugin base on the title of the site

#### What I expected to happen
I expected the atomic site's plugin page to load cleanly

#### What actually happened
Instead the plugin page is loaded but a query is sent for the title of the atomic site

#### Browser / OS version
Chromium Version 87.0.664.75 (Official build) (64-bit)

Windows 10

#### Is this specific to the applied theme? Which one?
No

#### Does this happen on simple or atomic sites or both?
Both

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Medium

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.
https://d.pr/i/HrjVCl

**Comment by getdave** [comment]
I've confirmed this happens when switching sites. Not sure why are this point but I get the following query string

`wp-admin/plugin-install.php?calypsoify=1&tab=search&s={{URL_OF_MY_ATOMIC_SITE}}`.

I tried switching `calypsoify` to `0` but that didn't change anything.

This appears to be a redirect.

A 2nd click on `Plugins` in the sidebar nav will land the user on the correct screen however so for now this is probably not a critical issue.

Let me know if you think I've got that wrong @jjchrisdiehl.
